total_scattering/autogrouping/similarity.py

```diff
@@ -0,0 +1,90 @@
+# code it
+# Bundle all of this into a class, rename it something better
+import numpy
+
+'''
+This file calculates similarity metrics for numpy vectors.
+It is build from cross correlations
+'''
+
+
+class similarity_metric:
```

Comment:
This class name breaks the [Python naming conventions](https://namingconvention.org/python/). Renaming to `SimilarityMetric` will fix this.